editoast/src/views/work_schedules.rs

```diff
@@ -13,25 +13,34 @@ use std::result::Result as StdResult;
 use thiserror::Error;
 use utoipa::ToSchema;
 
+use crate::core::pathfinding::TrackRange as CoreTrackRange;
 use crate::error::InternalError;
 use crate::error::Result;
 use crate::models::prelude::*;
 use crate::models::work_schedules::WorkSchedule;
 use crate::models::work_schedules::WorkScheduleGroup;
 use crate::models::work_schedules::WorkScheduleType;
+use crate::views::path::projection::PathProjection;
 use crate::views::AuthorizationError;
 use crate::views::AuthorizerExt;
 use crate::AppState;
-use editoast_schemas::infra::TrackRange;
+use editoast_schemas::infra::{Direction, TrackRange};
 
 crate::routes! {
-    "/work_schedules" => create,
+    "/work_schedules" => {
+        create,
+        "/project" => project,
```

Comment:
The existing endpoint for `train_schedule` is named `project_path`. It would be a good idea to keep the same naming. On top of that, we also have a project object in our model (in studies) so `project` alone can be misleading.

```suggestion
        "/project_path" => project,
```

Reply:
Here we are projecting a work_schedule on a path, it’s length of the path that’s defining the different intervals returned.
what do you think of `project_on_path` ?

Reply:
That's not a part of the job I know the most. Can you explain the difference with `train_schedule/project_path`? I thought they were for similar purposes.

Note that in any case, I believe that anything that is not just `project` will be better.

Reply:
`train_schedule/project_path` you project the path of one train on the path of another train (which is being displayed on the GET, y-axis is the length of the path)

here we are projecting a list of workschedulet on a path being displayed on the GET

Reply:
So both are projecting on the path of train. I'd say it'd be better to also keep consistency in the naming. So keep `project_path` for both. What do you think?

Reply:
alright

Reply:
done